### DISCUSSION TITLE
How to convert it to windows exe file?

### DISCUSSION THREAD

**Discussion by fbs2016** [python-gitlab/python-gitlab#2670] [General]
I can install python-gitlab by pip, and how to convert it to windows exe file as the host maybe not have python env. pyinstallser?

**Comment by nejch** [comment]
This is a general Python question and not really specific to python-gitlab.

But I have some experience with this so I can maybe point you in the right direction. A few tools can do this, the more popular ones being https://pyinstaller.org/en/stable/ and https://nuitka.net/index.html. Hope this helps! Let us know how it goes.

**Reply by fbs2016** [reply]
Thanks
